k8s/mongodb/mongo-ss.yaml

```diff
@@ -21,7 +21,7 @@ spec:
       terminationGracePeriodSeconds: 10
       containers:
       - name: mongodb
-        image: bigchaindb/localmongodb:1.0
+        image: bigchaindb/localmongodb:unstable
```

Comment:
See my other comment about the `unstable` bigchaindb/bigchaindb images. Do you think the `bigchaindb/localmongodb:unstable` image will change much?